configure.ac

```diff
@@ -684,6 +685,7 @@ case "${host}" in
 	if test "${LG_SIZEOF_PTR}" = "3"; then
 	  default_retain="1"
 	fi
+	zero_realloc_default_free="1"
```

Comment:
Verified that Scudo is doing the "free" option: https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/scudo/scudo_allocator.cpp#L720